docs/integrations/postcss.md

```diff
@@ -27,6 +27,18 @@ This package is in an experimental state right now. It doesn't follow semver, an
   ```
 :::
 
+```ts
+// nuxt.config.ts
+export default defineNuxtConfig({
+
+  css: ['~/assets/css/styles.css'], // add here path to your main css file
+  modules: [
+    '@unocss/nuxt',
+  ],
+})
+```
+:::
```

Comment:
Emm, no, postcss is a framework-agnostic plugin. We should not add this here as it can be confusing to users of other frameworks. Also I believe `@unocss/nuxt` is not the correct entry to use postcsee